### PR TITLE
Python Trie Github action

### PR DESCRIPTION
**Pull request by tullur**
Hello, @gmuraru 

1. Added a workflow for Python Trie.
2. Added test coverage with coverage report and also fail if coverage < 100%. Example: 
image

### CODE REVIEW

**Comment by gmuraru** [comment]
Done - merged my PR. The tests should pass

**Comment by gmuraru** [comment]
Done - merged my PR. The tests should pass

**Comment by gmuraru** [comment]
Also, could you remove python 3.5 and 3.6? :+1:

**Comment by tullur** [comment]
> Also, could you remove python 3.5 and 3.6? +1

Yep

**Comment by gmuraru** [comment]
Also, a heads-up - there is a commit linter

**Comment by gmuraru** [comment]
Q: In GitHub actions could you directly install a conda environment? (because there is an ```env.yaml``` file)

**Comment by tullur** [comment]
> Q: In GitHub actions could you directly install a conda environment? (because there is an `env.yaml` file)

Yes

File: .github/workflows/python_test.yml [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+        activate-environment: micendemons
+    - name: Conda info
+      shell: bash -l {0}
+      run: conda info
+    - name: Conda list
+      shell: pwsh
+      run: conda list
+    - name: Test with pytest
+      run: |
+        conda install pytest coverage
```
Comment (gmuraru) [inline .github/workflows/python_test.yml]: I think this can be removed

File: .github/workflows/python_test.yml [hunk 2]
```diff
@@ -0,0 +1,31 @@
...
+        $CONDA/bin/coverage run -m pytest Languages/Python
```
Comment (gmuraru) [inline .github/workflows/python_test.yml]: Can't we simply run ```coverage```?

Comment (tullur) [reply .github/workflows/python_test.yml]: Fixed

File: .github/workflows/python_test.yml (hunk 1 above)
Comment (tullur) [reply .github/workflows/python_test.yml]: Done

**Comment by gmuraru** [comment]
Thank you very much.

**Review by gmuraru — APPROVED** [review]

**Pull request merged**